packages/schematics/angular/ng-new/schema.json

```diff
@@ -130,8 +130,10 @@
     },
     "strict": {
       "description": "Creates a workspace with stricter type checking and stricter bundle budgets settings.",
+      "x-prompt": "Create a workspace with stricter type checking and stricter bundle budgets settings? [https://angular.io/guide/strict-mode]",
```

Comment:
I think we should also add something along the lines of `This is recommended to take full advantage of TypeScript and Angular language service features.`